contributing/contributing.md

```diff
@@ -76,3 +76,16 @@ you should reopen your terminal to find "(base)" next to your prompt: ![](conda_
     ```
 
 1. [Add a GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-new-gpg-key-to-your-github-account ) to your Modin account. Your commits need to be signed with a GPG key. For mac, you can use [Mac GPG](https://gpgtools.org/).
+
+
+1. (Optional) We recommend a few workflow settings:
+
+    1. If you use Visual Studio Code, auto-format with [black](https://black.readthedocs.io/en/stable/) every time you save changes:
+        1. Install [Microsoft's Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+        1. Open your VSCode settings, in `Code -> Preferences -> Settings`.
+        1. Search for "python formatting provider" and select "black" from the dropdown menu.
+        1. Again in settings, search for "format on save" and enable the "Editor: Format on Save" option.
+    2. Add a pre-commit hook:
+        1. In your modin repository, copy [this pre-commit file](pre-commit) to `.git/hooks/pre-commit`
+        1. Every time you try to commit, git will run flake8 and scripts/doc_checker.py and abort the commit if either fails. This lets you make sure your commits pass these tests before you commit them to GitHub.
```

Comment:
I do not see `scripts/doc_checker.py` run in pre-commit file.

Reply:
I deleted the reference to it here.